misc/packaging/apache/ansible-runner-service.spec

```diff
@@ -72,8 +72,8 @@ mkdir -p %{buildroot}%{_var}/www/runnner
 install -m 644 ./wsgi.py %{buildroot}%{_var}/www/runnner/runner.wsgi
 
 %post
-semanage fcontext -a -t httpd_log_t -s system_u /var/log/ovirt-engine/ansible-runner-service.log 2> /dev/null || semanage fcontext -m -t httpd_log_t -s system_u /var/log/ovirt-engine/ansible-runner-service.log
-[[ -f /var/log/ovirt-engine/ansible-runner-service.log ]] && restorecon -rF /var/log/ovirt-engine/ansible-runner-service.log
+semanage fcontext -a -t httpd_log_t -s system_u /var/log/ovirt-engine/ansible-runner-service.log 2> /dev/null || semanage fcontext -m -t httpd_log_t -s system_u /var/log/ovirt-engine/ansible-runner-service.log || true
+[[ -f /var/log/ovirt-engine/ansible-runner-service.log ]] && restorecon -rF /var/log/ovirt-engine/ansible-runner-service.log || true
```

Comment:
Same here, why should restorecon fail when there is existing log file?

Reply:
If the path does not exist it returns 1.
```cmd
[root@10-37-138-22 ~]# [[ -f /var/log/ovirt-engine/ansible-runner-service.log ]] && restorecon -rF /var/log/ovirt-engine/ansible-runner-service.log
[root@10-37-138-22 ~]# echo $?
1
```

Reply:
Same as above